deploy/cloudformation/static-host.yml

```diff
@@ -162,8 +155,30 @@ Resources:
                 Fn::Join: [':', [!Ref InfrastructureStackName, 'LogBucket']]
               - s3
               - !Ref AWS::URLSuffix
-          Prefix: !If
-            - NoFQDN
-            - !Sub web/${AWS::StackName}/
-            - !Sub web/${FQDN}/
+          Prefix: 
+            !Sub
```

Comment:
We've been putting ! functions inline with the property usually. So move this to the line above as `Prefix: !Sub`

Reply:
This and a number of other similar issues fixed in latest version.